README.md

```diff
@@ -8,7 +8,11 @@ This is a simple wrapper for the native iOS and Android Contact Picker UIs, with
 
 ```
 yarn add react-native-select-contact
-./node_modules/.bin/react-native link react-native-select-contact
+```
+
+For React Native => 0.59 only:
```

Comment:
Your PR description says "RN < 60", but this one says "RN >= 0.59"... which is correct?

Reply:
Both technically the same. It's a matter of preference. Seems like a lot of other packages are using "RN >= 0.59".

Reply:
Thank you.  FWIW, "RN < 60" means "react-native less than v60", and "RN >= 0.59" means "react-native at or above v59", so technically very different, but I appreciate the clarification!